src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -112,6 +140,13 @@ public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> customParseDate(String date) {
+        try {
+            DateTimeFormatter dateTimeFormatter
+                    = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.UK);
```

Comment:
make constants for all formatters)https://stackoverflow.com/questions/66066/what-is-the-best-way-to-implement-constants-in-java